types/module/server/router.go

```diff
@@ -193,3 +210,11 @@ func (rtr *router) testQueryFactory() InvokerFactory {
 		return rtr.invoker(callInfo.Method, nil)
 	}
 }
+
+func TypeURL(req proto.Message) string {
+	return fmt.Sprintf("/%s", proto.MessageName(req))
```

Comment:
```suggestion
	return "/%s" + proto.MessageName(req)
```

iirc there was a small perf PR that made this change in SDK repo, could do it here too

Reply:
Or we can use sdk.MsgTypeURL directly

Reply:
`sdk.MsgTypeURL` takes a `sdk.Msg` but here that's not limited to msgs